### PR TITLE
20200726 Sync docs English and Japanese

### PR DESCRIPTION
**Pull request by TakaoShimizu**
**What type of PR is this?**

/kind content

/kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

/area documentation

**What this PR does / why we need it**:
Sync docs English and Japanese.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
/assign @leodido
/assign @fntlnz

### CODE REVIEW

File: content/jp/docs/running.md [hunk 1]
```diff
@@ -1,153 +1,164 @@
...
+    -v /usr:/host/usr:ro \
+    -v /etc:/host/etc:ro \
+    falcosecurity/falco:latest
 ```
 
-4. FalcoをKind Kubernetesクラスター内に[Install](../installation)する。
+その他の設定可能なオプション:
 
-## Falcoを手動で実行する
+- DRIVER_REPO` - [ドライバのインストール](https://falco.org/docs/installation/#install-driver) を参照してください。
```
Comment (leogr) [inline content/jp/docs/running.md:155]:
```suggestion
- `DRIVER_REPO` - [ドライバのインストール](https://falco.org/docs/installation/#install-driver) を参照してください。
```

Just a missing backtick

[the remark this answers is not in the document]
Comment (TakaoShimizu) [reply content/jp/docs/running.md:155]: Thanks!

File: content/jp/docs/_index.md [hunk 2]
```diff
@@ -1,42 +1,86 @@
...
+ - Linux の `coreutils` 実行ファイルを変異させる
+ - ログインバイナリの変異 
+ - `shadowutil` や `passwd` の実行ファイルを変異させる  
+    - `shadowconfig`
+    - `pwck`
+    - `chpasswd`
+    - `getpasswd`
+    - `change`
+    - `useradd`
+    - etc
```
[the remark this answers is not in the document]
Comment (TakaoShimizu) [reply content/jp/docs/_index.md]: Thanks! deleted.

File: content/jp/docs/configuration.md [hunk 3]
```diff
@@ -1,13 +1,26 @@
...
 notoc: true
 ---
 
-Falcoの設定ファイルは、 'key:value'または'key:[value list]'ペアのコレクションを含む[YAML](http://www.yaml.org/start.html)ファイルです。
+{{< info >}}
 
-設定オプションはすべて、'-o/--option key=value'フラグを使用してコマンドラインで上書きできます。'key:[value list]'オプションの場合、'--option key.subkey=value'を使用して個々のリスト項目を指定できます。
+これはFalcoデーモンの設定オプションのためのものです。
+
+これらのオプションについては、[rules]（.../rules）または[alerts]（.../alerts）を参照してください。
```
[the remark this answers is not in the document]
Comment (TakaoShimizu) [reply content/jp/docs/configuration.md]: Thanks! updated!

File: content/jp/docs/running.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (TakaoShimizu) [reply content/jp/docs/running.md:155]: Thanks! added it!

**Pull request merged**